test/jdk/java/awt/Robot/MouseMoveOffScreen.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) 2024, Oracle and/or its affiliates. All rights reserved.
```

Comment:
```suggestion
 * Copyright (c) 2025, Oracle and/or its affiliates. All rights reserved.
```

It hasn't been integrated in 2024, so the year needs bumping up to 2025.